lib/experimental/kses.php

```diff
@@ -87,3 +87,22 @@ function allow_filter_in_styles( $allow_css, $css_test_string ) {
 }
 
 add_filter( 'safecss_filter_attr_allow_css', 'allow_filter_in_styles', 10, 2 );
+
+/**
+ * Mark CSS safe if it contains grid functions
+ *
+ * This function should not be backported to core.
+ *
+ * @param bool   $allow_css Whether the CSS is allowed.
+ * @param string $css_test_string The CSS to test.
+ */
+function allow_grid_functions_in_styles( $allow_css, $css_test_string ) {
+	if ( preg_match(
+		'/^grid-template-columns:\s*repeat\([0-9,a-z-\s\(\)]*\)$/',
```

Comment:
This looks good to me for now, but just wondering what the fix in core will be for it? I suppose we don't need to figure that out right now, but I was curious if it'll just be a matter of adding `repeat` to [this line](https://github.com/WordPress/wordpress-develop/blob/4a7009a1d2bc73c028c24b8824f71b2ec357ec52/src/wp-includes/kses.php#L2566) in core, or if it'll be more complex.

Reply:
Judging by the [previous update](https://core.trac.wordpress.org/changeset/54092) it should be pretty straightforward.